src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -1259,38 +1335,41 @@ def move(event_cb):
             mousepos = event.getPosition()
             ctrl = event.wasCtrlDown()
             shift = event.wasShiftDown()
-            self.pt = FreeCADGui.Snapper.snap(mousepos, lastpoint=last,
-                                              active=ctrl, constrain=shift)
-            if hasattr(FreeCAD, "DraftWorkingPlane"):
+            self.pt = Gui.Snapper.snap(mousepos, lastpoint=last,
+                                       active=ctrl, constrain=shift)
+            if hasattr(App, "DraftWorkingPlane"):
                 self.ui.displayPoint(self.pt, last,
-                                     plane=FreeCAD.DraftWorkingPlane,
-                                     mask=FreeCADGui.Snapper.affinity)
+                                     plane = App.DraftWorkingPlane,
+                                     mask = App.Snapper.affinity)
```

Comment:
These are `key=value` arguments inside a function. In this case, there should not be a space around the equals sign.
```py
functions(key=value, one="string")
```

The space is needed outside of function arguments.
```py
result = function()
b = second_f(key=result, another_key="some string")
```